apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -128,6 +129,32 @@ const PollWorkerScreen: React.FC<Props> = ({
     setIsPrintingPrecinctScannerReport,
   ] = useState(false)
 
+  useEffect(() => {
+    // If none of the modals are open, retrigger audiofocus
```

Comment:
Oof, this feels super likely to get out of date. Any good way to avoid that?

Reply:
Yeah, I considered just not doing this as this just makes the audio for "Poll Worker Actions" to repeat after the modals are closed. I think having it repeat when some of the modals are closed is still better then nothing even if future modals forget to add themselves here, but I'll add some more comments and rearrange the code a bit to hopefully make it more obvious to do this.